helpers/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  expectThrow: async promise => {
+    try {
+      await promise
+    } catch (error) {
+      // console.warn(error.message)
+      // TODO: Check jump destination to destinguish between a throw
+      //       and an actual invalid jump.
+      const invalidJump = error.message.search('invalid JUMP') >= 0
+
+      // TODO: Found out in which circumstances 'invalid opcode' is throwed
```

Comment:
Minor note, but it should be "Find out .... ".
Also, please remove this TODO per the previous comment.